app/controllers/concerns/caching_headers.rb

```diff
@@ -1,3 +1,4 @@
+# Included in ApplicationController for edge caching
```

Comment:
FWIW we explicitly disabled this in Rubocop:

```yml
Style/Documentation:
  Description: 'Document classes and non-namespace modules.'
  Enabled: false
```

I understand that disabling means makes them optional so adding them is ok. I prefer having them. I just wanted to point this out since `rubocop` is a tool we "officially" use whereas `reek` is not.

Reply:
I'm surprised reek only pointed out these few concerns modules (unless it's specifically looking for mixins and skipping classes) since adding class documentation would have produced a much larger diff.

I agree they usually add little value (but having a hint about where it might be included, and more importantly _why_, isn't a bad thing, and your comments include both a what and why here.

Reply:
I wasn't too worried about following reek's suggestions (it wants these docstrings literally everywhere), but I think it does make sense to include a comment on the concerns because they can jump into the app wherever and are sometimes hard to track down.